### ISSUE TITLE
can not uninstall from Apps and Features 

### ISSUE THREAD

**Issue by dvescovi1** [dorssel/usbipd-win#435]
when trying to uninstall from the main Windows "Apps and Features" pop up says "can't be done without Admin privileges" and stalls.

Uninstall from Admin power shell does work ok.

Uninstall using standard windows probably needs "elevated privilege" prompt.

**Comment by dorssel** [comment]
What OS version are you using? Is the user part of the administrator group (and is elevation sufficient), or is it a standard user (which would require a request for admin credentials)? Is the computer domain joined, and/or are other group policies in effect?

The installer works properly with UAC, also the uninstall. I've just re-confirmed on Windows 10 + Standard User: it correctly asks for admin credentials. Only if you cancel that prompt will the uninstaller fail with:

![admin-required](https://user-images.githubusercontent.com/17404029/186453993-cd9c22f8-43b4-471f-bc6b-a3cd3713caeb.png)

Is this the popup you see?
The prompt for credentials does *not* come from the uninstaller, but from the OS itself (probably msiexec.exe, which is part of the OS). The above popup does come from the uninstaller, but only after the OS already failed to get admin rights for the user.

TL;DR: I cannot reproduce the problem. I'm sure many people have uninstalled/re-installed at some point, and you are the unfortunate one to run into this. Maybe you are having OS problems as mentioned here: https://answers.microsoft.com/en-us/windows/forum/all/windows-10-cant-uninstall-programsuse-admin/34683993-295f-41ee-9ca5-03eda0f3afc7.
